torch/_subclasses/fake_tensor.py

```diff
@@ -2715,6 +2793,66 @@ def dump_cache_stats() -> None:
             log.info("    %-*s %s", width + 1, f"{k}:", v)
 
 
+def _make_fake(
```

Comment:
nit: could we call this "infer_fake_kernel_from_real" or something more descriptive? make_fake sounds like we're turning a real thing into a fake thing (which is close)